### PR TITLE
Adds new spacing tokens in service of Variables work

### PR DESCRIPTION
**Pull request by bruugey**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [Name of ticket](https://jira.mongodb.org/browse/[name-of-ticket])

### For bug fixes, new features & breaking changes

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have run `yarn changeset` and documented my changes

### For new components

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README
- [ ] I have verified the Live Example looks as intended on the design website.

### CODE REVIEW

File: packages/tokens/src/space.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const s000 = 0;
+export const s025 = 1;
+export const s050 = 2;
+export const s100 = 4;
+export const s150 = 6;
+export const s200 = 8;
+export const s300 = 12;
+export const s400 = 16;
```
Comment (TheSonOfThomp) [inline packages/tokens/src/space.ts:8]: do we want this file?

Comment (bruugey) [reply packages/tokens/src/space.ts]: Good catch, deleted.

**Review by shaneeza — COMMENTED** [review]
Thoughts on stacking the spacing tokens in the story so its easier to read?

**Comment by bruugey** [comment]
> Thoughts on stacking the spacing tokens in the story so its easier to read?

Updated!

**Comment by bruugey** [comment]
Screenshot 2024-01-29 at 12 46 48 PM

File: packages/tokens/src/Tokens.story.tsx [hunk 2]
```diff
@@ -70,6 +70,7 @@ export const Spacing = () => (
     <div
       className={css`
         display: flex;
+        flex-direction: column;
```
Comment (shaneeza) [inline packages/tokens/src/Tokens.story.tsx:73]: maybe we can add a gap, but it looks good!

**Review by shaneeza — APPROVED** [review]

File: .changeset/violet-buckets-lick.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/tokens': minor
+---
+
+Adds new tokens to spacing object, and deprecates (in name only) old tokens. Eventually, we will enforce a migration to the new system.
```
Comment (TheSonOfThomp) [inline .changeset/violet-buckets-lick.md:5]: What do you mean "in name only"?

**Pull request merged**